example/script.js

```diff
@@ -7,17 +7,25 @@ if (typeof qviz != 'undefined') {
     /* These examples shows how to draw circuits into containers. */
     const entangleDiv = document.getElementById('entangle');
     if (entangleDiv != null) {
-        qviz.draw(entangle, entangleDiv, qviz.STYLES['Default']);
+        qviz.draw(entangle, entangleDiv, qviz.STYLES['Default'], 0, true);
     }
 
     const sampleDiv = document.getElementById('sample');
     if (sampleDiv != null) {
-        qviz.draw(sample, sampleDiv, qviz.STYLES['Default']);
+        const isEditable = true;
+        const onCircuitChange = (circuit) => {
+            console.log('New circuit ↓');
+            console.log(circuit);
+        };
+        /* Pass in isEditable = true to allow circuit to be editable */
+        /* Pass in onCircuitChange to trigger callback function
+           whenever there is a change in circuit */
+        qviz.draw(sample, sampleDiv, qviz.STYLES['Default'], 0, isEditable, onCircuitChange);
     }
 
     const teleportDiv = document.getElementById('teleport');
     if (teleportDiv != null) {
-        qviz.draw(teleport, teleportDiv, qviz.STYLES['Default']);
+        qviz.draw(teleport, teleportDiv, qviz.STYLES['Default'], 0, true);
```

Comment:
It would be nice to show how to do this programmatically, for example, add a checkbox on the UI that enables/disable editing on the example.